templates/cloudflare-workers/wrangler.toml

```diff
@@ -1,12 +1,14 @@
+#:schema node_modules/wrangler/config-schema.json
```

Comment:
Where's this coming from?

Reply:
If you use an IDE extension that support json schema over TOML, this will give you auto completion and validation on your wrangler.toml, like [this one](https://marketplace.visualstudio.com/items?itemName=tamasfe.even-better-toml#completion-and-validation-with-json-schema).

Reply:
fabulous

Reply:
It's generated by the wrangler.

> Where's this coming from?